scripts/create_gitlab_snapshot.py

```diff
@@ -50,17 +50,20 @@ def gitlab_instance_info() -> Mapping[str, JSON]:
 
 def shutdown_instance(instance: Mapping[str, JSON]):
     instance_ids = [instance['InstanceId']]
-    log.info('Preparing to stop GitLab instance for %s, waiting 10 seconds before proceeding',
+    log.info('Preparing to stop GitLab instance for %s, waiting 10 seconds '
+             'before proceeding. Hit Ctrl-C to abort',
              config.deployment_stage)
     sleep(10)
     log.info('Stopping instance %s', instance_ids)
     aws.ec2.stop_instances(InstanceIds=instance_ids)
     try:
-        log.info('Waiting up to 10 minutes for the instance to stop')
+        log.info('Waiting for the GitLab instance to stop')
         waiter = aws.ec2.get_waiter('instance_stopped')
-        waiter.wait(InstanceIds=instance_ids)
+        waiter.wait(InstanceIds=instance_ids,
+                    WaiterConfig=dict(MaxAttempts=9999, Delay=15))
     except botocore.exceptions.WaiterError as e:
         log.warning(e)
+        exit(1)
```

Comment:
Just let the normal exception handling take care of the exit status. If the WaiterError's message isn't self-explanatory enough, raise a different exception with a message that is.